Makefile

```diff
@@ -1,6 +1,6 @@
 export GOOS=windows
 export DOCKER_IMAGE_NAME ?= windows-exporter
-export DOCKER_REPO ?= ghcr.io/prometheus-community
+export DOCKER_REPO ?= docker.io/prometheuscommunity ghcr.io/prometheus-community ghcr.io/prometheus-community
```

Comment:
Is this related to the PDH change? It's under one of the "fix lint" commits?

Reply:
no, wrong PR